install/installer/traf_start

```diff
@@ -58,19 +58,18 @@ else
        error1392=$(grep "1392" $SQ_ROOT/initializeErrors.txt | wc -l)
        error1395=$(grep "1395" $SQ_ROOT/initializeErrors.txt | wc -l)
        if [ "$error1395" -ge "1" ] || [ "$error1392" -ge "1" ]; then
-          ssh $STARTING_NODE "cd $SQ_ROOT/sql/scripts;  echo \"get version of metadata;\" | sqlci" | tee -a $SQ_ROOT/initalizeErrors.txt
+          ssh $STARTING_NODE "cd $SQ_ROOT/sql/scripts;  echo \"get version of metadata;\" | sqlci" | tee -a $SQ_ROOT/initializeErrors.txt
            metadataCurrent=$(grep "Metadata is current" $SQ_ROOT/initializeErrors.txt | wc -l)
            if [[ "$metadataCurrent" -ne "1" ]]; then
-              ssh $STARTING_NODE "cd $SQ_ROOT/sql/scripts;  echo \"initialize Trafodion, upgrade;\" | sqlci" | tee -a $SQ_ROOT/initializeErrors.txt
+              ssh $STARTING_NODE "cd $SQ_ROOT/sql/scripts;  echo \"initialize Trafodion, upgrade;\" | sqlci" | tee $SQ_ROOT/initializeErrors.txt
            fi
-        fi
-
-        if [ "$?" != "0" ]; then
+       fi
+       anyError=$(grep "ERROR" $SQ_ROOT/initializeErrors.txt | wc -l)
```

Comment:
Since the error file is being appended for upgrade, the errors txt file will contain 1395 failures. We want the install to proceed and not exit